.eslintrc

```diff
@@ -18,7 +18,7 @@
     },
     "extends": "eslint:recommended",
     "parserOptions": {
-        "ecmaVersion": 6,
+        "ecmaVersion": 10,
```

Comment:
Why this change, and why this version in particular? What's the minimum Node version do we want this library to require, and - since there's no build/transpile step - what ES version makes sense to lint as?

Reply:
eslint borked on `...mapping`. #138 raised Node to 10.x